earn/src/components/borrow/BorrowGraph.tsx

```diff
@@ -14,14 +14,33 @@ import BorrowGraphTooltip from './BorrowGraphTooltip';
 const TEXT_COLOR = '#82a0b6';
 const GREEN_COLOR = '#82ca9d';
 const PURPLE_COLOR = '#8884d8';
-const MILLIS_PER_WEEK = 10 * 24 * 60 * 60 * 1000;
+const MILLIS_PER_DAY = 24 * 60 * 60 * 1000;
+const DAYS_TO_SHOW = 60;
+const NUM_DATA_POINTS = 100;
 
 export type BorrowGraphData = {
   IV: number;
   'Collateral Factor': number;
   x: Date;
 };
 
+const interp = (a: BorrowGraphData, b: BorrowGraphData, x: number): BorrowGraphData => {
```

Comment:
1. Could we change this from an arrow function to a regular function?
2. Could we call it `interpolate` rather than `interp`? 
3. Could we use better variable names?
4. Could we add jsDoc explaining the function and the arguments?